modules/ROOT/pages/mergetags.adoc

```diff
@@ -0,0 +1,55 @@
+= Merge Tags plugin
+:navtitle: Merge Tags
+:description: Enables Merge Tags functionality in the {productname} editor
+:keywords: merge, tags, tag, mergetags, mergetag, mergetags_list, mergetags_prefix, mergetags_suffix, mail merge, personalization token
+:pluginname: Merge Tags
+:plugincode: mergetags
+:pluginminimumplan: tiertwo
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a merge tag (also known as a personalization token, or a mail merge field) from a drop-down list when a specified prefix is typed.
+
+Once the merge tag is inserted, the plugin leaves
+
+* a non-editable variable wrapped in a prefix; and
+* a suffix making the tag easily identifiable.
+
+== Interactive example
+
+liveDemo::merge-tag[]
+
+include::partial$misc/purchase-premium-plugins.adoc[]
+
+== Basic setup
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'mergetags',
+  toolbar: 'mergetags',
+});
+----
+
+The {pluginname} plugin provides an xref:autocompleter.adoc[autocompleter] for adding a merge tag without using the toolbar button or menu item.
```

Comment:
I assume that we are mentioning the autocompleter here for more technical people. Since the initial descriptions talks about a drop-down. So I just want to verify is that on purpose making the initial text less complex then dig into the details in lower down in the text or should both mention the autocompleter and possible link to that?

Reply:
> So I just want to verify is that on purpose making the initial text less complex then dig into the details in lower down in the text or should both mention the autocompleter and possible link to that?

This is deliberate, although it’s not really about about catering to the non-technical up-front and the technical further down.

It’s more the standard inverted pyramid or progressive reveal structure.

1. this is what we’re talking about.
2. this is what it does.
3. this is how it works (the example).
4. and here are even more details.

Reply:
Also, I incorporated your suggested change from a local edit that I pushed to the branch. Your diff accidentally included the *Suggested change* meta-text, so it was ever-so slightly quicker to do the edit locally and push compared to committing here, pulling, removing the meta text, and pushing.